### DISCUSSION TITLE
Cloud Hubs Address for Root Domain points to domain instead of app

### DISCUSSION THREAD

**Discussion by kala-goyal** [mozilla/hubs#2706] [General Questions & Discussion] [answered]
I'm trying to set up an instance of Cloud Hubs. I deployed everything, and am using Recipe 3 for the domains. On completion, though, the "Address for Root Domain" points to the domain instead of the other way around. The system admin running the domain told me that it should point to the app itself, and that if we enter in the Address into the CNAME as said here https://hubs.mozilla.com/docs/hubs-cloud-aws-existing-domain.html, we'll just end up in a circular point. 

Did I do something wrong, or will what pops up on the "Address for Root Domain" just change once the domain is connected?

**Comment by kala-goyal** [comment]
Does anyone have any ideas as to what is happening here?

**Comment by robinkwilson** [comment, marked as answer]
@kala-goyal Can you email **hubs@mozilla.com** with this question? I'd like to help, but need more information from you. I'll post the solution here when we work it out!

**Reply by kala-goyal** [reply]
We worked it out! It was a confusion of how the AddressForRootDomain would work-- the AWS expert at our organization said that typing in the AddressFor___ should have redirected to the app, and thus that there was something wrong, but once we updated the CNAME record, everything was resolved. Thank you though!

**Reply by robinkwilson** [reply]
Fantastic! Yes, that's in our docs for [Domain Recipe 3](https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html#recipe-3-domain-can-not-be-on-route-53) and [Using an existing domain](https://hubs.mozilla.com/docs/hubs-cloud-aws-existing-domain.html): the step after your stack completes DNS setup. 

I'll make it clearer in the documentation that it's an important step after the stack finishes.

**Reply by robinkwilson** [reply]
Done! Quick fix: https://github.com/MozillaReality/hubs-docs/commit/07ad10c24b5a61d09f651660d6e54ef4cf8eb260
